components/Modal/Modal.js

```diff
@@ -28,28 +28,54 @@ function Modal({
   isOpen,
   onRequestClose,
   screenReaderLabel,
-  shouldCloseOnOverlayClick,
+  canClose,
   childrenClassName,
 }) {
   if (isOpen) {
     gtag.modalView(screenReaderLabel);
   }
 
+  const portalContainer =
```

Comment:
Hey @subhajit20 ! I dove into this one rather than spend more of your time than you bargained for...

The issue where the modal was not rendering is explained because we use `'#__next'` in the ["important" config](https://tailwindcss.com/docs/configuration#important). Radix's Dialog component uses React Portal to render as a direct child of `body` rather than the div with the `#__next` ID (as screenshotted). We needed to specifiy the `container` prop on the `Dialog.Portal` component so that Tailwind's styles actually applied!

![Screenshot 2023-09-23 at 11 38 34 PM](https://github.com/OperationCode/front-end/assets/9523719/34522d0c-e569-4217-93de-090451a5da3f)

I made some other changes, but nothing you need to really worry about! Great work again.